crates/net/network/src/peers/manager.rs

```diff
@@ -277,15 +277,15 @@ impl PeersManager {
 
```

Comment:
we will want to increment this after we checked if the peer passes trusted check. at best, the trusted check is moved out of the match on entry variant, up here, since it is the same regardless if the entry is occupied or vacant.
```rust
self.connection_info.inc_in();
```